src/main/java/seedu/address/model/date/Date.java

```diff
@@ -8,12 +8,12 @@
 import java.time.format.DateTimeParseException;
 
 /**
- * Represents a Person's date of birth in the address book.
+ * Represents a date in the application.
  * Guarantees: immutable; is valid as declared in {@link #isValidDate(String)}
  */
-public class DateOfBirth implements Comparable<DateOfBirth> {
+public class Date implements Comparable<Date> {
 
-    public static final String MESSAGE_CONSTRAINTS = "Date of birth must be in format: dd/mm/yyyy";
+    public static final String MESSAGE_CONSTRAINTS = "Dates must be in format: dd/MM/yyyy";
```

Comment:
Understand that internally it must be represented as MM when use DateTimeFormatter as mm represents minutes, but since this is a user-facing message, I think it'll be better if we continue to use lowercase so as not confuse them with the technicalities of the backend. What do ya think? 

```suggestion
    public static final String MESSAGE_CONSTRAINTS = "Dates must be in format: dd/mm/yyyy";
```

Reply:
Yup I agree! Thanks for pointing it out!